esmvaltool/diag_scripts/monitor/multi_datasets.py

```diff
@@ -292,6 +298,47 @@
     coordinates. Can be adjusted to avoid overlap with the figure. Only
     relevant if ``show_stats: true``.
 
+
+Configuration options for plot type ``1d_profile``
+--------------------------------------------------
+plot_bias: string, optional
+    Optional switch to choose whether difference plot to reference should
+    be applied. And if plot_bias is not False or empty, it defines if
+    difference is 'absolute' or 'relative'.
```

Comment:
I would recommend not doing this in the diagnostic but rather use the [`bias`](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/recipe/preprocessor.html#id36) preprocessor. That way the code for the diagnostic will be much simpler.

Reply:
Yes, you should be able to use the [`convert_units`](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/recipe/preprocessor.html#convert-units) for this. Note that you also need to specify [`custom_order: true`](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/recipe/overview.html#recipe-section-preprocessors) in your preprocessor since in the [default order](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/api/esmvalcore.preprocessor.html#preprocessor-functions) `bias` is executed after `convert_units`.

Reply:
Thank you, good to know. Then I will delete the bias calculation in the 1d_profile.